extra/datasets/kits19.py

```diff
@@ -126,6 +160,73 @@ def sliding_window_inference(model, inputs, labels, roi_shape=(128, 128, 128), o
   result = result[..., paddings[4]:image_shape[0]+paddings[4], paddings[2]:image_shape[1]+paddings[2], paddings[0]:image_shape[2]+paddings[0]]
   return result, labels
 
+def rand_flip(image, label, axis=(1, 2, 3)):
+  prob = 1 / len(axis)
+  for ax in axis:
+    if random.random() < prob:
+      image = np.flip(image, axis=ax).copy()
+      label = np.flip(label, axis=ax).copy()
+  return image, label
+
+def random_brightness_augmentation(image, factor=0.3, prob=0.1):
+  if random.random() < prob:
+    factor = np.random.uniform(low=1.0-factor, high=1.0+factor, size=1)
```

Comment:
> can move 1+ into low and high.

I created `low` and `high` instead as arguments.

> use python random if you just need one

I switched over to it and re-ran the test and it failed. I believe this is because of the fact that [numpy.random.uniform](https://numpy.org/doc/stable/reference/random/generated/numpy.random.uniform.html) excludes the upper bound, while the python version doesn't.

> but is [2-0.3, 2+0.3] correct?

interestingly, the reference implementation had these values set as these: `[1-0.3, 1+0.3]`.

Reply:
meh it's fine if it matches reference